lib/project_types/script/forms/push.rb

```diff
@@ -6,13 +6,24 @@ class Push < ScriptForm
       flag_arguments :api_key, :force
 
       def ask
-        self.api_key ||= ask_api_key
+        if ScriptProject.current.env.nil?
+          org = ask_org
+          app = ask_app(org)
+          self.api_key = app['apiKey']
+          write_env(JSON.generate(org), api_key, app['apiSecretKeys'].first['secret'], nil)
```

Comment:
- I'm wondering if we should avoid serializing the org, what if we store the org_id and continue to fetch the org on each command instead? I'm worried about if the org query changes. There are also multiple org queries in use already
- Why are we storing `app['apiSecretKeys'].first['secret']` in the env now? I've never seen that before